cocos/rendering/custom/builtin-forward-pipeline.ts

```diff
@@ -292,6 +300,24 @@ function setupCameraConfigs (
     const isEditorView: boolean = camera.cameraUsage === CameraUsage.SCENE_VIEW || camera.cameraUsage === CameraUsage.PREVIEW;
     cameraConfigs.enablePostProcess = pipelineConfigs.useFloatOutput && camera.usePostProcess && (isMainGameWindow || isEditorView);
     cameraConfigs.enableProfiler = DEBUG && isMainGameWindow;
+    cameraConfigs.pipelineSettings = camera.pipelineSettings;
+
+    if (isEditorView) {
+        cameraConfigs.pipelineSettings = getEditorPipelineSettings();
+        if (cameraConfigs.pipelineSettings) {
+            const pipelineCamera: Camera | null = getEditorPipelineCamera();
+            cameraConfigs.enablePostProcess = pipelineConfigs.useFloatOutput
+                && pipelineCamera !== null && pipelineCamera.usePostProcess;
+        } else {
+            cameraConfigs.enablePostProcess = false;
+        }
+    }
```

Comment:
This is to support preview pipeline effect in `SceneView` or `Preview` window in creator, to achieve WYSIWYG.